tests/s3_api_gradle.erl

```diff
@@ -1,33 +1,54 @@
 -module(s3_api_gradle).
--include_lib("eunit/include/eunit.hrl").
 
 -behavior(riak_test).
+
+-include_lib("eunit/include/eunit.hrl").
+
 -export([
     confirm/0
 ]).
 
 -define(NODE_COUNT, 3).
+-define(TEST_USER, ["test", "test@example.com"]).
+-define(METADATA_KEY, [{<<"security">>, <<"users">>}, <<"test">>]).
 
 confirm() ->
-    Nodes = rt:deploy_nodes(?NODE_COUNT),
-    ok = rt:wait_until_nodes_ready(Nodes),
+    %% Build a simple cluster
+    Nodes = rt:build_cluster(?NODE_COUNT),
+
+    %% Choose random node to interact with
+    Node = lists:nth(random:uniform(?NODE_COUNT), Nodes),
 
-    %% Set to testing profile backend on all the nodes
-    {[ok, ok, ok], []} = rpc:multicall(Nodes,
-                                       application,
-                                       set_env,
-                                       [riak_s3_api, default_user_backend, riak_s3_user_profile_backend]),
+    %% Create test user and wait for it to appear on the cluster
+    _ = rpc:call(Node, riak_s3_user, create, ?TEST_USER),
+    rt:wait_until(fun() ->
+                        {Results, []} = rpc:multicall(Nodes, riak_core_metadata, get, ?METADATA_KEY),
+                        case lists:filter(fun(R) -> R == undefined end, Results) of
```

Comment:
Unless I'm misunderstanding something, you really want to do something like:
```erlang
lists:all(fun(R) -> R =/= undefined end, Results)
```
instead of the case statement here... you want to wait until all nodes have the metadata key you're looking for, and none of them return undefined, right?  As it stands now, it'll return `false` if there are no records that are undefined (which means keep waiting) but `true` otherwise (when there are still undefined results)?